### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by meichthys**
- Update chat link to link to Matrix space instead of single room
- Emphasize Awesome Trilium repo
- Remove Ukraine images
- Add note about links still referring to original repo
- Emojify all headings
- Add "Why TriliumNext" section

### CODE REVIEW

**Comment by Byteingpython** [comment]
Could You maybe move the link to the wiki? It doesn't really fit into a where to discuss section. Could you als maybe either mark these links to other languages as old or maybe translate the new README?

File: README.md [hunk 1]
```diff
@@ -1,33 +1,21 @@
...
-
-## Discuss with us
-
-Feel free to join our discussions.
-
-- [XMPP](https://joinjabber.org/): [xmpp:discuss@trilium.thisgreat.party?join](xmpp:discuss@trilium.thisgreat.party?join) ([web link](https://anonymous.cheogram.com/discuss@trilium.thisgreat.party))
-- [Matrix](https://matrix.org/try-matrix/): #trilium:matrix.org ([web link](https://app.element.io/#/room/#trilium:matrix.org))
-
-The two rooms are mirrored, so you can use the protocol of your choice, from the client you prefer, on pretty much any platform under the sun!
+<a href="https://github.com/zadam/trilium/wiki/Screenshot-tour"><img src="https://raw.githubusercontent.com/wiki/zadam/trilium/images/screenshot.png" alt="Trilium Screenshot" width="1000"></a>
```
Comment (Alumniminium) [inline README.md:7]: Might as well include the screenshots and tour in this repo and not link to zadams.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,21 @@
-# Trilium Notes
-
-## Trilium is in maintenance mode - see details in https://github.com/zadam/trilium/issues/4620
-
-Preliminary disccusions on the successor organization are taking place in [Trilium Next discussions](https://github.com/orgs/TriliumNext/discussions). 
+# TriliumNext Notes
 
 [English](https://github.com/zadam/trilium/blob/master/README.md) | [Chinese](https://github.com/zadam/trilium/blob/master/README-ZH_CN.md) | [Russian](https://github.com/zadam/trilium/blob/master/README.ru.md) | [Japanese](https://github.com/zadam/trilium/blob/master/README.ja.md) | [Italian](https://github.com/zadam/trilium/blob/master/README.it.md)
```
Comment (Alumniminium) [inline README.md:3]: Still links to the OG repo

**Review by Alumniminium — APPROVED** [review]

**Review by CobriMediaJulien — APPROVED** [review]
Didnt find any faults :)

**Review by JYC333 — APPROVED** [review]

**Pull request merged**